prow/cluster/prow-canary-k8s-io_managedcertificate.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: networking.gke.io/v1beta1
+kind: ManagedCertificate
+metadata:
+  name: prow-canary-k8s-io
+spec:
+  domains:
+    - prow-canary.k8s.io
```

Comment:
wrong indent?

Reply:
It is the indentation my editor uses when not otherwise constrained! (and the indentation I use in other contexts.)

Will fix.